play/psycopg3/README.md

```diff
@@ -0,0 +1,6 @@
+# Example using TAIL with psycopg3
+
+This example script shows how to create an asynchronous consumer of a Materialized View using the
+psycopg3 driver. You will need to follow the instructions for installing psycopg3 from source.
+
+To see the script in action, call the script as so: `./async_tail.py VIEW_NAME`.
```

Comment:
I think mentioning that this works best with a demo of some sort -- having complete instructions to run this with no other context would help.

So, for example:

```markdown
To see the script in action, create a view that has streaming data. If you don't have your own data the billing demo is a good place to start.

From the root of the repo run:

    BILLING_MESSAGE_COUNT=1000000 bin/mzconduct run billing --workflow local

This will create a variety of views, but an interesting one is `billing_monthly_statement`.

You can then invoke this script and point it at any view, using the monthly statement mentioned above:

    play/psycopg3/async_tail.py billing_monthly_statement

See the [detailed tail documentation](https://materialize.com/docs/sql/tail/) for a guide on interpreting the results.
```